tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/XUnit/XUnitIntegration.cs

```diff
@@ -142,6 +130,7 @@ internal static void FinishScope(Scope scope, IExceptionAggregator exceptionAggr
             }
 
             scope.Dispose();
+            Common.StopCoverage(scope.Span);
```

Comment:
I think the chances of this actually being required are very slim, but would it be wise to surround these two lines with a `finally` block?

Reply:
Same question applies to the other integrations as well.

Reply:
True, done in [7bd0800](https://github.com/DataDog/dd-trace-dotnet/pull/2969/commits/7bd0800faad3e6e2c1b72ad01677bebd03975744)